src/main/java/org/itsallcode/junit/sysextensions/security/ExitGuardSecurityManager.java

```diff
@@ -46,11 +55,233 @@ public void checkPermission(final Permission perm)
     /**
```

Comment:
Do you need to also delegate the `checkPermission()` and `checkExit()` methods?

Reply:
Good point.
Fixed.